opba-api-security/src/main/java/de/adorsys/opba/api/security/internal/filter/RequestSignatureValidationFilter.java

```diff
@@ -82,6 +76,36 @@ protected boolean shouldNotFilter(HttpServletRequest request) {
         return !matcher.match(OPBA_BANKING_PATH, pathHelper.getPathWithinApplication(request));
     }
 
+    private boolean verifyRequestSignature(HttpServletRequest request, FilterValidationHeaderValues headerValues, Instant instant, String fintechApiKey) {
+        HttpRequestToDataToSignMapper mapper = new HttpRequestToDataToSignMapper();
+        String operationType = headerValues.getOperationType();
+        boolean verificationResult;
+
+        switch (operationType) {
+            case "AIS":
```

Comment:
Can we use enum-based constants?

Reply:
Done, enum values are used instead of string literals